cdap-data-fabric/src/test/java/co/cask/cdap/data2/dataset2/lib/partitioned/PartitionedFileSetTest.java

```diff
@@ -116,6 +120,63 @@ public void testDecodeIncomplete() {
   }
 
   @Test
+  public void testPartitionMetadata() throws Exception {
```

Comment:
ok, well, for now, updated by wrapping the actions of a test case within just 1 transaction, and pushed.